### PR TITLE
Issue 18 57 61 84 86 89 95 97

### PR DESCRIPTION
**Pull request by tae898**
This PR tackles 4 issues (actually 3)

* https://github.com/intelligent-systems-course/schnapsen/issues/18
* https://github.com/intelligent-systems-course/schnapsen/issues/86
* https://github.com/intelligent-systems-course/schnapsen/issues/95
* https://github.com/intelligent-systems-course/schnapsen/issues/97

It's a bit messy since this PR tackles more than one issue, but it better be done before the IS assignments start.
Michael and I aim to get this done by the end of 6-Nov-2023.

### CODE REVIEW

**Comment by tae898** [comment]
This is not done yet. Once it's done, I'll ping Michael

**Comment by tae898** [comment]
I haven't found a code issue for https://github.com/intelligent-systems-course/schnapsen/issues/18.

File: tests/bots/test_randbot.py
Comment (miselico) [inline tests/bots/test_randbot.py:1]: I don't see how this is a test. This just let the bot play against itself, but nothing is checked.

File: tests/bots/test_rdeepbot.py
Comment (miselico) [inline tests/bots/test_rdeepbot.py:1]: I don't see how this is a test. This just let the bot play against itself, but nothing is checked.

Comment (miselico) [reply tests/bots/test_rdeepbot.py:1]: Left as an issue in #21

File: tests/bots/test_randbot.py
Comment (miselico) [reply tests/bots/test_randbot.py:1]: Left as an issue in #21

**Comment by miselico** [comment]
Closes #18 
Closes #61 
Closes #86 
Closes #95 
Closes #97

**Review by miselico — APPROVED** [review]

**Comment by miselico** [comment]
Ready for merging. @tae898 can you do a sanity check?

**Pull request merged**